### DISCUSSION TITLE
Suggestion: Move the location of the "Apply Metadata" button above the Metadata block.

### DISCUSSION THREAD

**Discussion by NemukaPrime** [lllyasviel/Fooocus#3557] [Ideas]
When wanting to apply an image's metadata, I have to scroll all the way to the bottom to apply it.
I think the button should be on top of the Metadata block, right under the "For images created by Fooocus" area. 

This is in case the user doesn't need to read the metadata and just wants to apply it. Thank you.

**Comment by GodwinF** [comment]
I agree or even better make it possible so you can drag the image to the prompt box and clicking the generate button will load all the meta data. Just how Forge works.
